src/core/multiphase.jl

```diff
@@ -0,0 +1,7 @@
+function wrapto180(degrees)
+    return degrees - 360*floor((degrees + 180)/360)
+end
+
+function wraptopi(radians)
+    return radians - 2*pi*floor((radians+pi)/(2*pi))
+end
```

Comment:
Unless they are used elsewhere, I would put these in `io/opendss` or `io/common` for now.

Reply:
ok